PyInstaller/hooks/rthooks/pyi_rth_multiprocessing.py

```diff
@@ -86,3 +92,27 @@ def __init__(self, *args, **kw):
                     os.putenv('_MEIPASS2', '')
 
 forking.Popen = _Popen
+
+# fix spawn as well
+# Patch Popen to re-set _MEIPASS2 from sys._MEIPASS.
+if not sys.platform.startswith('win'):
+    class _Spawning_Popen(spawning.Popen):
+        def __init__(self, *args, **kw):
+            if hasattr(sys, 'frozen'):
+                # We have to set original _MEIPASS2 value from sys._MEIPASS
+                # to get --onefile mode working.
+                os.putenv('_MEIPASS2', sys._MEIPASS)  # @UndefinedVariable
```

Comment:
Please remove the linter comment.
```suggestion
                os.putenv('_MEIPASS2', sys._MEIPASS) 
```